drivers/hud_apr/app/models/hud_apr/generators/shared/fy2021/base.rb

```diff
@@ -52,7 +52,8 @@ def needs_ce_assessments?
           household_assessment_required[hh_id] = annual_assessment_expected?(hoh_enrollment)
           date = [
             @report.start_date,
-            last_service_history_enrollment.first_date_in_program,
+            # Only HoHs get CE assessments, so we use their entry date
+            needs_ce_assessments? ? hoh_enrollment.first_date_in_program : last_service_history_enrollment.first_date_in_program,
```

Comment:
minor, but can we use `if then else end` instead of `? :`  I have a much harder time parsing it.